### PR TITLE
MINOR: Improvements to release.py

### PR DESCRIPTION
**Pull request by omkreddy**
- Update release.py to trim "-SNAPSHOT" from docs/js/templateData.js while adding "Bump version commit" during release process. 
- Add blog entry details to announcement_email template

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -348,6 +348,9 @@ def command_release_announcement_email():
 
 <DETAILS OF THE CHANGES>
 
+An overview of the release and its notable changes can be found in the
+release blog post: <LINK TO THE KAFKA BLOG>
```
Comment (chia7712) [inline release.py]: Can we generate the link automatically if the link formate is like `https://kafka.apache.org/blog#apache_kafka_362_release_announcement`

**Review by chia7712 — APPROVED** [review]
LGTM, and only one comment left.

File: release.py (hunk 1 above)
Comment (omkreddy) [reply release.py]: Thanks for the review,. updated the PR

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**